src/Slices/Notification/UI/Drawer/Item.tsx

```diff
@@ -23,18 +22,31 @@ interface Props {
 }
 
 export const Item: React.FC<Props> = ({ notification, onUpdate }) => {
+  const { routeManager } = useContext(DependencyContext);
+  const detailsLink: RouteKindWithId<"CompileDetails"> | undefined =
+    routeManager.getUrlCompileDetailsId(
+      notification.uri
+    ) as RouteKindWithId<"CompileDetails">;
+  const navigate = useNavigateTo();
+
+  const onClick = (notification: Notification): void => {
```

Comment:
This `notification` is the same as the one received in the `props` of this component, so no need to pass it to the function.